src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square implements Figure {
+    private String color;
+    private double side;
+
+    public Square(String color, double side) {
+        this.color = color;
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Square: color="
```

Comment:
Magic numbers or strings should be avoided. The string "Square" should be defined as a constant to avoid repetition and to make future changes easier. For example, `private static final String FIGURE_TYPE = "Square";`